Intro_to_Supervised_Learning.md

```diff
@@ -0,0 +1,36 @@
+# Intro to Supervised Learning by, Jeffrey Ugochukwu
+
+## What is Supervised Learning?
+       
+Hello, fellow reader. I see that you're very interested in finding interesting applications of using Statistics to solve many
```

Comment:
Jeffrey, I like what you were trying to do with making this blog more personable for the reader. I would recommend removing phrases that directly target the reader like "Hello, fellow reader" and "you" (or lessing the number of times you use them).